test/Unit/PowerShell/PowerShellManagerTests.cs

```diff
@@ -149,6 +149,13 @@ public void ModulePathShouldBeSetCorrectly()
             string workerModulePath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             string funcAppModulePath = Path.Join(FunctionLoader.FunctionAppRootPath, "Modules");
             string expectedPath = $"{funcAppModulePath}{Path.PathSeparator}{workerModulePath}";
+
+            if (Platform.IsWindows)
+            {
+                string latestAzModulePath = @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules\1.0.0";
+                expectedPath = $"{funcAppModulePath}{Path.PathSeparator}{workerModulePath}{Path.PathSeparator}{latestAzModulePath}";
+            }
```

Comment:
This whole check can be removed, as `latestAzModulePath` will never be set in the environment variable even on Windows unless `WEBSITE_INSTANCE_ID` is set (which is not set when running the tests).
This caused the test to fail in CI.